### PR TITLE
Add support for Glassfish 7.0.1

### PR DESCRIPTION
**Pull request by pepness**
- Add missing validations for servlet 5 and 6
- Add missing glassfish 4 objects in ASDDVersion class

NetBeans Testing:

- Verify successful execution of libraries and licenses Ant test
- Verify successful execution of Verify Sigtests
- Verify successful execution of unit tests for modules glassfish.common, glassfish.javaee, glassfish.tooloing, glassfish.eecommon and j2ee.sun.dd
- Started NetBeans and ensure the log didn't have any ERROR or new WARNINGS
- Successfully register GlassFish 7.0.1, create a web app and verify that it works.

### CODE REVIEW

File: enterprise/glassfish.javaee/src/org/netbeans/modules/glassfish/javaee/Hk2JavaEEPlatformFactory.java [hunk 1]
```diff
@@ -71,6 +71,10 @@ public class Hk2JavaEEPlatformFactory extends J2eePlatformFactory {
      *  when changing it. */
     private static final String V6_LOOKUP_KEY = "J2EE/DeploymentPlugins/gfv6ee9/Lookup";
     private static final String V610_LOOKUP_KEY = "J2EE/DeploymentPlugins/gfv610ee9/Lookup";
+    
+    /** GlassFish V7 JakartaEE platform lookup key.
+     *  <p/>We will keep V3 value now because no one knows what will get broken
+     *  when changing it. */
```
Comment (matthiasblaesing) [inline enterprise/glassfish.javaee/src/org/netbeans/modules/glassfish/javaee/Hk2JavaEEPlatformFactory.java:77]: ?!

**Review by matthiasblaesing — APPROVED** [review]
Looks sane to me (not an expert here though), I saw an unclear comment that may need some more explanation.

File: enterprise/glassfish.javaee/src/org/netbeans/modules/glassfish/javaee/Hk2JavaEEPlatformFactory.java (hunk 1 above)
Comment (pepness) [reply enterprise/glassfish.javaee/src/org/netbeans/modules/glassfish/javaee/Hk2JavaEEPlatformFactory.java]: Done. There is no need for that comment anymore.

**Review by matthiasblaesing — APPROVED** [review]
Thank you for the update

**Pull request merged**